ale/base/label_pds3.py

```diff
@@ -61,26 +61,19 @@ def target_name(self):
         : str
           target name
         """
-
         return self.label['TARGET_NAME']
 
     @property
     def _target_id(self):
         return spice.bodn2c(self.label['TARGET_NAME'])
 
     @property
-    def starting_ephemeris_time(self):
-        if not hasattr(self, '_starting_ephemeris_time'):
-            sclock = self.label['SPACECRAFT_CLOCK_START_COUNT']
-            self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
-        return self._starting_ephemeris_time
+    def _starting_ephemeris_time(self):
+        return self.label['SPACECRAFT_CLOCK_START_COUNT']
```

Comment:
It makes sense to have the clock count be provided by the label and then the SPICE mix-in can convert it to an ephemeris time.

Reply:
With the current naming, the scs2e calls in SPICE don't make sense, it looks like converting ephemeris time to ephemeris time with a method that converts clock strings to ephemeris time.

Reply:
agree that at least better naming is needed. Not sure what though.

Reply:
`spacecraft_clock_start_count` seems like the logical choice to me.